satpy/cf/__init__.py

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 """Code for generation of CF-compliant datasets."""
+
+EPOCH = u"seconds since 1970-01-01 00:00:00"
```

Comment:
I don't have a better suggestion necessarily, but can this go somewhere else? It is the only thing in this module and it just seems odd and easy to misplace/forget about. I can be convinced otherwise.

Reply:
I moved it to satpy.cf.coords where it's actually used !